charts/internal/calico/templates/node/daemonset-calico-node.yaml

```diff
@@ -173,10 +173,29 @@ spec:
             requests:
               cpu: 10m
               memory: 50Mi
+          volumeMounts:
+            - mountPath: /run/xtables.lock
+              name: xtables-lock
+              readOnly: false
```

Comment:
Nit: Usually the `volumeMounts` are at the end of the container specification. Not really a problem, but I was just a bit surprised.